packages/bitcore-wallet-service/test/integration/server.js

```diff
@@ -10757,19 +10757,22 @@ describe('Wallet service', function() {
       });
 
       describe('User logged out', () => {
-        it('should return swap crypto disabled if the user is located in NY', () => {
-          const opts = {
-            currentLocationCountry: 'US',
-            currentLocationState: 'NY',
-          };
-    
-          server.getServicesData(opts, (err, config) => {
-            should.not.exist(err);
-            should.exist(config.swapCrypto);
-            config.swapCrypto.disabled.should.equal(true);
-            config.swapCrypto.disabledMessage.should.equal('Swaps are currently unavailable in your area.');
+        const usaBannedStates = ['HI', 'LA', 'NY'];
+        usaBannedStates.forEach(bannedState => {
```

Comment:
Please use `for (const bannedState in usaBannedStates) {` instead of .forEach. forEach is slow and a for loop is easier to step through with a debugger